Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -449,14 +503,38 @@ const ScrollView = React.createClass({
       };
     }
 
+    const hasStickyHeaders = this.props.stickyHeaderIndices && this.props.stickyHeaderIndices.length > 0;
+
+    const children = hasStickyHeaders ?
+      React.Children.toArray(this.props.children).map((child, index) => {
```

Comment:
I did try using React.Children.map but it doesn't work, looks like indices are different. My guess is that toArray handles null / nested children differently and that's the behavior we want here.